apps/entry/widgets/conditional_widget.py

```diff
@@ -52,9 +53,9 @@ def update_attribute(widget, data, widget_data):
 
         if w_export_data:
             excel_data.append(w_export_data.get('data', {}).get('excel'))
-            report_keys += w_export_data.get('data', {})\
-                .get('report', {})\
-                .get('keys') or []
+            report_data = w_export_data.get('data', {})\
+                .get('report', {})
+            report_keys = report_data.get('keys') or []
```

Comment:
There is `+=` before?

Reply:
I found it was not necessary. `report_keys = []` was directly used just there.